functions/src/index.ts

```diff
@@ -2,14 +2,22 @@ import * as functions from 'firebase-functions';
 import * as admin from 'firebase-admin';
 import { accountType } from './services/user/getAccountType';
 import { User } from './models/user.model';
+// import serviceAccount from './dashboard-dev.json'; // Replace the firebase json file here
```

Comment:
Is this comment still needed?

Reply:
@eddiejaoude I have added this comment for your reference.

Reply:
Ok, thank you, can you please add a `@TODO` so I remember to look at it